pkg/storage/store.go

```diff
@@ -124,6 +124,15 @@ var importRequestsLimit = settings.RegisterPositiveIntSetting(
 	1,
 )
 
+// addSSTableRequestMaxRate is the maximum number of AddSSTable requests per second.
+var addSSTableRequestMaxRate = settings.RegisterPositiveIntSetting(
+	"kv.bulk_io_write.addsstable_max_rate",
+	"maximum number of AddSSTable requests per second for a single store",
+	math.MaxInt64,
+)
+
+const addSSTableRequestBurst = math.MaxInt64
```

Comment:
This should probably be lowered to something that's feasible to actually hit. Even if the cluster starts with a low rate limit, if a schema change starts after after the cluster has been up for a long time, I think the token bucket in the rate limiter will still have MaxInt64 tokens, so no rate limiting will actually take effect (unless we write MaxInt64 SSTs). Not sure what a reasonable limit would be, though.